README.adoc

```diff
@@ -1,33 +1,83 @@
-== Empty Repository
+= AsciiDoc für Software-Architektur Workshop
+:toc:
+:icons: font
+:source-highlighter: highlight.js
 
-This is just an empty repository to play around with some Diagrams-as-Code tools.
+== Über diesen Workshop
 
-To open this repository in gitpod, just preface the URL with `gitpod.io#`.
+Dieser Workshop führt dich in die Dokumentation von Software-Architektur mit AsciiDoc und docToolchain ein. Du lernst, wie du das arc42-Template verwenden, Architekturdiagramme erstellen und Architekturentscheidungen dokumentieren kannst.
 
-Next, gitpod will start a web based visual studio instance with everything pre-installed.
+Die Übungen bauen aufeinander auf und vermitteln praktische Fähigkeiten für die Erstellung professioneller Architekturdokumentation.
 
-Open this `README.adoc` and press `Ctrl+k` then `v` to aktivate the AsciiDoc preview.
+== Voraussetzungen
 
-Now open xref:src/docs/diagrams-as-code.adoc[] and start hacking.
+* Grundlegende Kenntnisse in der Softwareentwicklung
+* Git-Grundlagen
+* Ein Texteditor mit AsciiDoc-Unterstützung (VSCode mit AsciiDoc-Plugin empfohlen)
```

Comment:
Die Idee ist den Workshop in einem Github Workspace zu starten. Damit wird ein VS Code im Browser geöffnet und somit seht ein Editor mit AsciiDoc-Preview zur Verfügung.

Reply:
Du solltest bitte erklären, dass der Workshop in einem Github Workspace zu starten ist.